geode-wan/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
   implementation(project(':geode-serialization'))
   implementation(project(':geode-tcp-server'))
   implementation(project(':geode-core'))
+  implementation(project(':geode-gfsh'))
```

Comment:
it's unfortunate that the _interface_ `GfshCommand` is in the same module as the gfsh _implementation_, but since it is, I see now why you have to import geode-gfsh from geode-wan.

Reply:
Looks like `geode-gfsh` is the owner of the Command interface needed by `geode-wan`. Makes sense as far as that goes.